.github/NIGHTLY_TEST_FAILURE.md

```diff
@@ -1,11 +0,0 @@
----
-title: "nightly test-integration failed"
```

Comment:
Why are we deleting this?

Reply:
The nightly builds no longer raise an issue if they fail so this is unused.